frontend-react/src/pages/admin/value-set-editor/ValueSetsIndex.test.tsx

```diff
@@ -101,7 +101,11 @@ describe("ValueSetsIndex tests", () => {
                 } as ValueSetsMetaResponse)
         );
         mockUseValueSetsTable = jest.fn(() => {
-            throw new RSNetworkError("Test", { status: 404 } as AxiosResponse);
+            throw new RSNetworkError(
+                new AxiosError("Test", "404", undefined, {}, {
+                    status: 404,
+                } as AxiosResponse)
+            );
         });
         /* Outputs a large error stack...should we consider hiding error stacks in page tests since we
          * test them via the ErrorBoundary test? */
```

Comment:
We'd considered this in a previous iteration of the team, and determined running tests with `--silent` was an okay patch for the time. Would love to bring this back up in norming @stephenkao!

Reply:
Hmmmm!  I thought I added a ticket for this a while back, but I'd love it if all our test warnings/errors were resolved.  It's not a huge deal since they're, ya know, in tests, but they can sometimes be indicative of erroneous rendering/state updates or incorrect mocks.

This is out of the scope of this changeset, though, since I don't think Joseph added this comment!  Unless GitHub is borking out again 😆